utils/generate_env.sh

```diff
@@ -0,0 +1,42 @@
+# sh ./generate_env.sh <dev|uat|prod>
+
+ENV=$1
+
+if [ -z "$ENV" ]
+then
+  echo "No environment specified. Usage: sh ./generate_env.sh <dev|uat|prod>"
+  exit 1
+fi
+
+
+subscription_dev="bbe47ad4-08b3-4925-94c5-1278e5819b86"
+subscription_uat="26abc801-0d8f-4a6e-ac5f-8e81bcc09112"
+subscription_prod="b9fc9419-6097-45fe-9f74-ba0641c91912"
+
+resource_group_dev="pagopa-d-weu-bizevents-rg"
+resource_group_uat="pagopa-u-weu-bizevents-rg"
+resource_group_prod="pagopa-p-weu-bizevents-rg"
+
+account_dev="pagopa-d-weu-bizevents-ds-cosmos-account"
+account_uat="pagopa-u-weu-bizevents-ds-cosmos-account"
+account_prod="pagopa-p-weu-bizevents-ds-cosmos-account"
+
+
+if [ "$ENV" = "dev" ]; then
```

Comment:
use ENV check only to set right variables subscription, resource and account and not copy&paste the same command